environments/change_to_relative_pos.py

```diff
@@ -0,0 +1,39 @@
+import argparse
+import numpy as np
+from os.path import join
+import shutil
+
```

Comment:
useful ?

Reply:
Yes, I still use this to change absolute pos to relative pos (because the real data is collected with absolute pos)

Reply:
okay can be used for old data, but setting is_discrete=True was enough I think (test_env.py).